scripts/sp-setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+az account show
+RESULT_LOGIN=$?
+if [ "$RESULT_LOGIN" != 0 ]; then
+  az login
+fi
+ACCOUNT_DETAILS=$(az account show)
+TENANT_ID=$(echo "$ACCOUNT_DETAILS" | grep "tenantId" | sed -rn "s/\"tenantId\": \"(.*)\",/\1/p" | xargs)
+SUBSCRIPTION_ID=$(echo "$ACCOUNT_DETAILS" | grep "id" | sed -rn "s/\"id\": \"(.*)\",/\1/p" | xargs)
+echo "Tenant is:$TENANT_ID"
+echo "Subscription is:$SUBSCRIPTION_ID"
+
+RESULT_SP_CREATE=$(az ad sp create-for-rbac --name "$SERVICE_PRINCIPAL_NAME")
+# shellcheck disable=SC2181
```

Comment:
Looks like you have been linting via shellcheck locally.  Can we add another separate PR that includes a `shellcheck` job in the `push` workflow.  (This may require some cleaning of ciaran's pre-existing shell scripts).

Reply:
@sharkinsspatial This should already be in here, but as part of one of my other PRs. It is a bit of a quagmire (giggidy) at the moment as I have 5 or 6 PRs open.

Reply:
👍  😄  Yup.  Just saw it as I was stepping through.